configs/a64fx.yaml

```diff
@@ -134,71 +135,71 @@ Execution-Units:
     - SVE_DIV_OR_SQRT
   1:
     Pipelined: True
-    Blocking-Groups: 
+    Blocking-Groups:
```

Comment:
Remove un-necessary formatting changes throughout

Reply:
I am unsure of whether these changes actually exist. Looking at git blame this line hasn't been changed for a year. I think the white space should be removed anyway; this is consistent with all of the other configs